deploy/README.md

```diff
@@ -1,58 +1,88 @@
-	sed "s@AWS_ACCESS_KEY_ID@\"${AWS_ACCESS_KEY_ID}\"@g" | \
-	sed "s@AWS_SECRET_ACCESS_KEY@\"${AWS_SECRET_ACCESS_KEY}\"@g" | \
-	sed "s@AWS_DEFAULT_REGION@\"${AWS_DEFAULT_REGION}\"@g" | \
-	sed "s@AWS_AVAILABILITY_ZONES@\"${AWS_AVAILABILITY_ZONES}\"@g" | \
-	sed "s@OSC_ACCOUNT_ID@\"${OSC_ACCOUNT_ID}\"@g" | \
-	sed "s@OSC_ACCOUNT_IAM@\"${OSC_ACCOUNT_IAM}\"@g" | \
-	sed "s@OSC_USER_ID@\"${OSC_USER_ID}\"@g" | \
-	sed "s@OSC_ARN@\"${OSC_ARN}\"@g" > apply_secret.yaml
-	
-	cat apply_secret.yaml
-	
-	/usr/local/bin/kubectl delete -f apply_secret.yaml --namespace=kube-system
-	/usr/local/bin/kubectl apply -f apply_secret.yaml --namespace=kube-system
-```
-
-## Deploy the CCM deamonset
-
-### Helm Configuration
-See [Helm Chart Configuration](../docs/helm.md)
-### Setps
-
-```
-	#Using the manifest
-	kubectl delete -f deploy/osc-ccm-manifest.yml
-	kubectl apply -f deploy/osc-ccm-manifest.yml
-	kubectl get pods -o wide -A -n kube-system | grep osc-cloud-controller-manager
-
-	#Using helm3
-	cd deploy
-	# set the IMAGE_SECRET, IMAGE_NAME, IMAGE_TAG, SECRET_NAME to the right values on your case
-	IMAGE_NAME=outscale/cloud-provider-osc && \
-	IMAGE_TAG=v0.0.10beta && \
-	SECRET_NAME=osc-secret && \
-	helm uninstall k8s-osc-ccm 
-	helm install k8s-osc-ccm deploy/k8s-osc-ccm \
-		--set oscSecretName=$SECRET_NAME \
-		--set image.repository=$IMAGE_NAME \
-		--set image.tag=$IMAGE_TAG
-		
-	kubectl get pods -o wide -A -n kube-system | grep osc-cloud-controller-manager
+This documentation explains how to deploy Outscale Cloud Controller Manager.
 
+# Prerequisites
+
+You will need a Kubernetes cluster on 3DS Outscale cloud. The next sections details prerequisites on some cloud resources.
+
+| Plugin Version | Minimal Kubernetes Version | Recommended Kubernetes Version |
+| -------------- | -------------------------- | ------------------------------ |
+| <= v0.0.10beta | 1.20                       | 1.23                           |
+
+# Configuration
+
+## Cluster Resource Tagging
+
+You must tag some cloud resources with a cluster identifier in order to allow Cloud Controller Manager to identify which resources are part of the cluster.
+This include:
+- [VPC](https://docs.outscale.com/en/userguide/About-VPCs.html)
+- [Instances](https://docs.outscale.com/en/userguide/About-Instances.html)
+- [Security Groups](https://docs.outscale.com/en/userguide/About-Security-Groups-(Concepts).html)
+- [Route Tables](https://docs.outscale.com/en/userguide/About-Route-Tables.html)
+
+The tag key must be `OscK8sClusterID/my-cluster-id` (adapt `my-cluster-id`) and tag value can be one of the following values:
+- `shared`: resource is shared between multiple clusters, and should not be destroyed
+- `owned`: the resource is considered owned and managed by the cluster
+
+## Instances Tagging
+
+Additionally, instances must be tagged with their node name.
+
+Tag key is `OscK8sNodeName` and tag value `kubernetes.io/my-kybernetes-host-name` (adapt `my-cluster-id`).
```

Comment:
The value is just `my-kybernetes-host-name` which should be the same as `kubernetes.io/hostname` computed